docs/essentials/gc/garbage/weakreference.md

```diff
@@ -1 +1,44 @@
+# Weak References
```

Comment:
Wouldn't it be better if this article talked about the generic `WeakReference<T>` instead of `WeakReference`?